frontend/src/config/data-provider-parameters.tsx

```diff
@@ -396,24 +396,23 @@ export const DATA_PROVIDER_PARAMETERS: { [k: string]: DataProviderParameter } =
     },
   };
 
-export const DATA_PROVIDER_WITH_ENROLLMENTS_IN_PRODUCTION_ENV = [
-  'franceconnect',
-  'api_entreprise',
-  'api_particulier',
-  'cartobio',
-  'api_impot_particulier_sandbox',
-  'api_impot_particulier_production',
-  'api_r2p_sandbox',
-  'api_r2p_production',
-  'api_impot_particulier_fc_sandbox',
-  'api_impot_particulier_fc_production',
-  'le_taxi',
-  'api_droits_cnam',
-  'api_ficoba_sandbox',
-  'api_ficoba_production',
-  'aidants_connect',
-  'hubee_portail',
-  'hubee_portail_dila',
-  'api_pro_sante_connect',
-  'api_tiers_de_prestation',
+export const HIDDEN_DATA_PROVIDER_WITH_ENROLLMENTS_IN_PRODUCTION_ENV = [
```

Comment:
I do not think the '_WITH_ENROLLMENTS_IN_PRODUCTION_ENV' makes much sense in this new variable name.

We should simplify to 'HIDDEN_DATA_PROVIDER_LABELS'